xml/MAIN-SBP-scomusermanual.xml

```diff
@@ -1571,7 +1571,7 @@ sles11sp2x64-om2012-sm17:/ #
                             <para>For creating the configuration channels and adding configuration
                                 files in the &suma; Web UI, please refer to the &suma;
                                 Installation and Troubleshooting Guide at <link
-                                    xlink:href="https://www.suse.com/documentation/suse-manager-3/singlehtml/suse_manager21/book_susemanager_install/book_susemanager_install.html"
+                                    xlink:href="https://documentation.suse.com/external-tree/en-us/suma/3.2/susemanager-getting-started/html/book.suma.getting-started/book.suma.getting-started.html"
```

Comment:
This sounds like it is the wrong book. Are you sure here?

Reply:
The Installation and Troubleshooting Guide does not exist anymore. The correct book is the Advanced Topics guide https://documentation.suse.com/external-tree/en-us/suma/3.2/susemanager-advanced-topics/html/book.suma.advanced.topics/book.suma.advanced.topics.html
And the text needs to be changed to "For creating the configuration channels [...], please refer to the Advanced Topics guide."

Getting Started is the correct book, but the text of course is wrong now and should be "